### ISSUE TITLE
Search by category

### ISSUE THREAD

**Issue by dimpflmoser** [scharel/harbour-nextcloudnotes#38]
In list view miss the option to filter notes  by their category

**Comment by scharel** [comment]
Yes, thats right. If you have an idea on how to realize this in the UI, please tell me.

Until this is realized you can already enter the category in the search field. But this will also show results that contain the category in the content.

**Comment by dimpflmoser** [comment]
How about to add syntax to search, such as category:myentry would only filter on categories.

When you click on the search field you could present the categories such as when you assign a note to a category

**Comment by scharel** [comment]
Yes, presenting a "search by" area when the searchfield is active should be doable.
